eth/tracers/api_test.go

```diff
@@ -421,6 +431,18 @@ func TestTraceTransaction(t *testing.T) {
 	}) {
 		t.Error("Transaction tracing result is different")
 	}
+
+	// test TraceTransaction for a historical transaction
+	result2, err := api.TraceTransaction(context.Background(), common.HexToHash("0xACDC"), nil)
+	resBytes, _ := json.Marshal(result2)
+	have2 := string(resBytes)
+	if err != nil {
+		t.Errorf("want no error, have %v", err)
+	}
+	want2 := `[{"result":"0x8888"}]`
+	if have2 != want2 {
+		t.Errorf("test result mismatch, have\n%v\n, want\n%v\n", have2, want2)
+	}
```

Comment:
I don't love the use of `result2`, `want2`, `have2` etc, but it got the job done. 
Open to suggestions here.

Reply:
I'm fine with this